js/ui/pivot_grid/ui.pivot_grid.field_chooser.js

```diff
@@ -180,7 +180,19 @@ var FieldChooser = BaseFieldChooser.inherit({
                  * @default 'All Fields'
                  */
                 allFields: messageLocalization.format("dxPivotGrid-allFields")
-            }
+            },
+            /**
+            * @name dxPivotGridFieldChooserOptions_applyChangesMode
+            * @publicName applyChangesMode
+            * @type string
+            * @default "instantly"
+            */
```

Comment:
Accepted values are missed.
It is better using an enum instead of string type in this case.